### ISSUE TITLE
Regression on Export/Download of Files introduced in 4.9.1

### ISSUE THREAD

**Issue by jennycalendar** [globaleaks/GlobaLeaks#3216]
**Describe the bug**
On selecting single/multiple reports and clicking "Export", I receive a popup message which says "Error!". When I look at the network tab, I can see that I'm getting an error "Method Not Implemented". Likewise, when I attempt to download an attachment on the report, it opens a new tab, which notifies me that I'm "Not Authenticated" with an error code of 10. 

**To Reproduce**
1. Log in as recipient
2. Click on reports tab
3. Select one or multiple records
4. Click export 
5. Error appears
6. Click on single report
7. Navigate to the attached files
8. Click Download
9. New tab opens with error.

**Log Details**

```
 - - [15/Apr/2022:21:03:22 +0000] "POST /api/token HTTP/1.1" 201 127 0ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36"
- -[15/Apr/2022:21:03:22 +0000] "GET /api/rfile/cdb5480d-2122-4ada-8eff-9ed2f0d6f303?token=9b56570fc8fa7bb929d862a75cc94fc3829cf8dd36671af083bcea9b0f962c58 HTTP/1.1" 412 82 0ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET / HTTP/1.1" 200 1804 1ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:05:51 +0000] "GET /css/styles.min.css HTTP/1.1" 200 53361 26ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /js/scripts.min.js HTTP/1.1" 200 259617 90ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /api/public HTTP/1.1" 200 2886 1ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /data/favicon.ico HTTP/1.1" 200 5703 6ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /lib/js/locale/angular-locale_en.js HTTP/1.1" 200 955 1ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /l10n/en HTTP/1.1" 200 9513 4ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:05:51 +0000] "GET /webfonts/metropolis-all-700-normal.woff2 HTTP/1.1" 200 26456 1ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /webfonts/metropolis-all-400-normal.woff2 HTTP/1.1" 200 24180 2ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:05:51 +0000] "GET /data/logo.png HTTP/1.1" 200 6869 2ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:05:51 +0000] "GET /webfonts/fa-solid-900.woff2 HTTP/1.1" 200 154296 26ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:06:00 +0000] "POST /api/token HTTP/1.1" 201 127 0ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:06:05 +0000] "POST /api/authentication HTTP/1.1" 201 196 4254ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:06:05 +0000] "GET /data/favicon.ico HTTP/1.1" 200 5703 4ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:06:05 +0000] "GET /api/preferences HTTP/1.1" 200 508 13ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36"
]- - [15/Apr/2022:21:06:07 +0000] "GET /data/favicon.ico HTTP/1.1" 200 5703 11ms - "[REMOVED_USER_AGENT]"
 - - [15/Apr/2022:21:06:07 +0000] "GET /api/preferences HTTP/1.1" 200 508 27ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36"
 - - [15/Apr/2022:21:06:07 +0000] "GET /api/rtips HTTP/1.1" 200 873 41ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36"
 - - [15/Apr/2022:21:06:10 +0000] "POST /api/token HTTP/1.1" 201 126 0ms - "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36"
- - [15/Apr/2022:21:06:10 +0000] "GET /api/rtips/133b72f7-b854-4db2-8dcc-4c7b966cc1f8/export?token=1f402e0a739e1065d61c98b640fb87a02c1ac3f81aea3f6e3d81d5d34c637779 HTTP/1.1" 412 82 0ms - "[REMOVED_USER_AGENT]"
- - [15/Apr/2022:21:06:15 +0000] "POST /api/rtips/133b72f7-b854-4db2-8dcc-4c7b966cc1f8/export HTTP/1.1" 501 85 -1ms - "[REMOVED_USER_AGENT]"
```

**Expected behavior**
Successful export of record or download of file.

**Desktop (please complete the following information):**
 - Ubuntu 20.08, accessing the recipient dashboard from Macbook iOS 12.0.1

**Comment by jennycalendar** [comment]
issue reproduced for me on try.globaleaks.com

**Comment by evilaliv3** [comment]
Thank you @jennycalendar

I confirm that this is a regression introduced latest release 4.9.1 with commit: https://github.com/globaleaks/GlobaLeaks/commit/7cb5aa5b035abd8c2fc178e7cab48d16b1d4d47f

**Comment by evilaliv3** [comment]
Fixed in https://github.com/globaleaks/GlobaLeaks/commit/6f24a32c0dba568bfbfc5ad53066337161b35ca9
Released in 4.9.2

Thank you again @jennycalendar
